TrafficCapture/captureProtobufs/build.gradle

```diff
@@ -9,11 +9,11 @@ plugins {
 
 dependencies {
     api project(":commonDependencyVersionConstraints")
-    api group: 'com.google.protobuf', name: 'protobuf-java', version: '3.22.2'
+    api group: 'com.google.protobuf', name: 'protobuf-java', version: '3.25.5'
 }
 
 protobuf {
     protoc {
-        artifact = "com.google.protobuf:protoc:3.22.2"
+        artifact = "com.google.protobuf:protoc:3.25.5"
```

Comment:
Can we try removing the version from here? I'm less sure about this one

Reply:
If you remove one of these, we really should remove the other too.  I want the version specifiers to be specified together in the same file.